opentelemetry-sdk/src/opentelemetry/sdk/propagator/b3_format.py

```diff
@@ -102,12 +107,15 @@ def extract(cls, get_from_carrier, carrier):
 
     @classmethod
     def inject(cls, context, set_in_carrier, carrier):
-        sampled = (trace.TraceOptions.RECORDED & context.trace_options) != 0
+        span_context = context[Tracer.CONTEXT_SLOT_NAME]
```

Comment:
Similar error here: span_context is actually a Span, you need to first retrieve the SpanContext from it.